alert_group_nl_log2slack.py

```diff
@@ -396,6 +406,9 @@ def fetch_logs_and_publish_forever():
                 send_slack_message(str(record))
                 print(f'sent message: {record}')
 
+        if HEALTH_FILE != '':
+            os.utime(HEALTH_FILE)
+
         time.sleep(300)
```

Comment:
Maybe the example shouldn't fail after 60s but after 600s?

Reply:
Good catch! changed it to 900.